modules/nw-osp-pod-creating-ipv6.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
```

Comment:
@MaysaMacedo @itzikb-redhat All good with the state of this module? 

(Preview: https://53860--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/network-configuration.html#nw-osp-pod-creating-ipv6_post-install-network-configuration)